tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/model/BarberPreferences.java

```diff
@@ -31,4 +31,23 @@ public BEARD_KIND getBeardTrimming() {
     return beardTrimming;
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
override of equals and hashcode has to be a best practice, since it reduces significantly the creation of dynamic configs/connections